internal/static/i18n/de.yaml

```diff
@@ -883,12 +936,140 @@ EventTypes:
           deactivated: Twilio SMS Provider deaktiviert
   key_pair:
     added: Schlüsselpaar hinzugefügt
+    certificate:
+      added: Zertifikat hinzugefügt
   action:
     added: Aktion hinzugefügt
     changed: Aktion geändert
     deactivated: Aktion deaktiviert
     reactivated: Aktion reaktiviert
     removed: Aktion gelöscht
+  instance:
+    added: Instanz hinzugefügt
+    changed: Instanz gelöscht
+    customtext:
+      removed: Kundenspezifischer Text gelöscht
+      set: Kundenspezifischer Text gelöscht
+      template:
+        removed: Kundenspezifische Textvorlage gelöscht
+    default:
+      language:
+        set: Standard Sprache gesetzt
+      org:
+        set: Standard Organisation gesetzt
+    domain:
+      added: Domain hinzugefügt
+      primary:
+        set: Primäre Domain gesetzt
+      removed: Domain gelöscht
+    iam:
+      console:
+        set: ZITADEL Console Applikation gesetzt
+      project:
+        set: ZITADEL Projekt gesetzt
+    mail:
+      template:
+        added: E-Mail Vorlage hinzugefügt
+        changed: E-Mail Vorlage geändert
+      text:
+        added: E-Mail Text hinzugefügt
+        changed: E-Mail Text geändert
+    member:
+      added: Instanzmitglied hinzugefügt
+      changed: Instanzmitglied geändert
+      removed: Instanzmitglied gelöscht
+      cascade:
+        removed: Instanzmitglieder kaskadierend gelöscht
+    notification: 
+      provider:
+        debug:
+          fileadded: Datei zu Debug Notification Provider hinzugefügt
+          filechanged: Datei von Debug Notification Provider geändert
+          fileremoved: Datei von Debug Notification Provider gelöscht
+          logadded: Log von Debug Notification Provider hinzugefügt
+          logchanged: Log von Debug Notification Provider geändert
+          logremoved: Log von Debug Notification Provider gelöscht
+    oidc:
+      settings:
+        added: OIDC Einstellung hinzugefügt
+        changed: OIDC Einstellung geändert
+    policy:
+      domain:
+        added: Domain Richtlinie hinzugefügt
+        changed: Domain Richtlinie geändert
+      label:
+        activated: Label Richtlinie aktiviert
+        added: Label Richtlinie hinzugefügt
+        assets:
+          removed: Asset von Label Richtlinie gelöscht
+        changed: Label Richtlinie geändert
+        font:
+          added: Schriftart zu Label Richtlinie hinzugefügt
+          removed: Schriftart von Label Richtlinie gelöscht
+        icon:
+          added: Symbol zu Label Richtlinie hinzugefügt
+          removed: Symbol von Label Richtlinie gelöscht
+          dark:
+            added: Symbol zu dunkler Label Richtlinie hinzugefügt
+            removed: Symbol von dunkler Label Richtlinie gelöscht
```

Comment:
we currently don't translate icon and have `(dunkler Modus)`, e.g. `Icon (dunkler Modus) zu Label Richtlinie hinzugefügt`, which I prefer